### ISSUE TITLE
errors in .bashrc: bash version issue? 

### ISSUE THREAD

**Issue by alexlenail** [ohmybash/oh-my-bash#330]
Coming from zshrc, thought I'd give this a go. I believe I'm ssh'd into a centos6 machine. 
```
$ echo "$BASH_VERSION"
4.1.2(1)-release
$ source .bashrc
-bash: declare: -g: invalid option
declare: usage: declare [-aAfFilrtux] [-p] [name[=value] ...]
-bash: declare: -g: invalid option
declare: usage: declare [-aAfFilrtux] [-p] [name[=value] ...]
-bash: declare: -g: invalid option
declare: usage: declare [-aAfFilrtux] [-p] [name[=value] ...]
-bash: declare: -g: invalid option
declare: usage: declare [-aAfFilrtux] [-p] [name[=value] ...]
-bash: : syntax error: operand expected (error token is "")
```

Seems like somewhere there's a `declare -g` and maybe a `:` where this version of bash isn't expecting one...?

**Comment by akinomyoga** [comment]
Thanks for the report! I have pushed a fix to `master` ed5a250. Could you check if it works?

**Comment by alexlenail** [comment]
Works!
